pcm-player.js

```diff
@@ -69,7 +70,7 @@ PCMPlayer.prototype.createContext = function() {
     this.audioCtx = new (window.AudioContext || window.webkitAudioContext)();
     this.webAudioTouchUnlock(this.audioCtx).then(function () {
         this.gainNode = this.audioCtx.createGain();
-        this.gainNode.gain.value = 1;
+        this.gainNode.gain.value = this.option.gain;
```

Comment:
would be nice to rename this member variable to `options` and add some method docs to explain the acceptable range of values

Reply:
@vocoded Done